.github/workflows/CI.yml

```
@@ -1025,3 +1025,45 @@ jobs:
         run: |
           grep -Lr "SPDX-License-Identifier: Apache-2.0" --exclude-dir="toolchain" --exclude-dir="install" --exclude-dir=".git" . --exclude-dir="third_party" --exclude-dir="TEST_*" --exclude-dir="runtime" | grep ".*\.h$" || [[ $? == 1 ]]
         shell: bash
+
+  generate-network-type-inference:
+      runs-on: ${{ needs.select-docker-image-and-runner.outputs.runner }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)